.github/scripts/backwards_compatibility_check.sh

```diff
@@ -67,7 +67,12 @@ find_removed_methods() {
   then
     echo "New minor release is being performed. Ignoring changes in classes marked with @KinesisClientInternalApi annotation."
   fi
-  local latest_classes=$(jar tf $LATEST_JAR | grep .class | tr / . |  sed 's/\.class$//')
+  local latest_classes=$(
+    jar tf $LATEST_JAR |
+    grep .class |
+    tr / . |  sed 's/\.class$//' |
```

Comment:
nit: move `tr / . |` and `sed 's/\.class$//' |` on different lines for consistency